specification/glossary.md

```diff
@@ -2,10 +2,130 @@
 
 *This section is non-normative.*
 
-`TODO` Needs to be completed and linked
+## Adjustment
 
-Resource:
-> Add definition
+A charge representing a modification to billing data to account for certain events or circumstances not previously captured, or captured incorrectly. Examples include: billing errors, service disruptions, or pricing changes.
 
-Service:
-> Add definition
+## Amortization
+
+The gradual recognition of upfront costs over the duration of time that the good or service is consumed. An upfront payment made toward a commitment based discount can be amortized over commitment life. Amortization is valuable when the commitment period (time duration of the cost) extends beyond the granularity of the source report.
+
+## Billed Cost
```

Comment:
@mike-finopsorg Is this going to conflict with the anchors that are generated for the columns? (Same for any other definitions that have the same names as these headers?)

Reply:
Here's what we need to add: 
<a class="glossary" name="glossary:billedcost">&nbsp;</a>

Im not sure if the class glossary will do anything. We may need to override that in our main css if needed. Based on a quick look, I'm not sure where the class for the standard links from the table of contents (class="toc-anchor") is coming from. For now, go with this please.